UI/menu/expanding.html

```diff
@@ -37,7 +37,10 @@
 					target = '_top'
 				<?lsmb END ?>
 				id="a_<?lsmb- item.id -?>"
-                                class="<?lsmb IF item.module; 'menu-terminus';
+                                class="<?lsmb IF item.label == 'New Window';
```

Comment:
see line 42/45 below.  <?lsmb text(item.label) ?> is where it is translated (so on display).